apps/authentication-service/src/resources/clients/clientsRoutes.ts

```diff
@@ -17,22 +17,42 @@ export const clientsRoutes = (app: Application): void => {
 
   clientsRouterV1.post(
     "/keys",
-    [validateAccessToken, requiredScopes(WidgetHostPermissions.CREATE_KEYS)],
+    [
+      validateAccessToken,
+      requiredScopes(
+        (WidgetHostPermissions as Record<string, string>).CREATE_KEYS,
+      ),
```

Comment:
I was getting linting errors on this, unless I specified the type for the `WidgetHostPermissions` enum. Very strange, but I gave up trying to figure out why.

The same goes for the other uses of `WidgetHostPermissions` in this file.

Reply:
This sounds like there's something wrong with the way the package is setup with typescript

Reply:
We should fix this. These shared enums aren't that useful if we don't know their types.

Reply:
Are you using the Just In time strategy? https://turbo.build/repo/docs/core-concepts/internal-packages#just-in-time-packages

Reply:
That strange thing is that in other places in the code where these are used, the typing isn't an issue. It was only in that one file, where I run into eslint complaining.

I'll look at the JIT stuff you linked to. I'm not sure if I'm using it.

Reply:
I just looked the JIT link. The one complication I can see, is that our node services aren't compiled, so they don't use webpack, turbopack, or vite, which is what that article suggests is necessary for JIT to work.

I'll do some more digging to see how I might be able to do this with node services.